packages/core/src/statistics/lidoStatistics.ts

```diff
@@ -12,7 +12,7 @@ export async function getLidoStatistics(this: SolidoSDK) {
   ]);
 
   return {
-    apy,
+    stakeApy: stakeApy.max,
```

Comment:
Why `stakeApy`? I understand that it would be a major versions
But is this change really necessary?

Reply:
because apy field now have this type:
```ts
type ApyData = {
  apy: number;
  apr: number;
  intervalPrices: {
    beginEpoch: number;
    endEpoch: number;
  };
};
```

So, it's weird to see code like this: `apy.apy`, or `apy.apr`. If you have a better idea, go.

Reply:
Or maybe even return not all structure, just like before, return only max value?

Reply:
Oh, u ar right, got it
maybe something like `yield`?

Reply:
But I think we can also return one value, cause clints could orient on values in intervalPrices that could differ

Reply:
done